website/blog/2023-01-25-ferretdb-fetches-data-query-pushdown.md

```diff
@@ -6,9 +6,13 @@ description: "Find out how FerretDB uses query pushdown to fetch data from the s
 image: /img/blog/ferretdb-query-pushdown.jpg
 ---
 
+Find out how FerretDB uses query pushdown to fetch data from the storage layer (also called as “backend”)
```

Comment:
```suggestion
Find out how FerretDB uses query pushdown to fetch data from PostgreSQL.
```
I think we have a lot of vocab to mean the same thing, _handler_, _storage layer_ and I see _backend_ too. But for this blog, it is specific to PosgreSQL so let's go with that. :hugs:

Reply:
Thanks @chilagrow